Delete member

simonisfiction opened the pull request:
I was using the wrong table view so the feature should have worked all  along lol

simonisfiction committed:
re implement the table model for members

re implemented the table model with QSqlTableModel instead of QSqlQueryModel. this gives me more flexibility

simonisfiction committed:
add an enum for member table headers

simonisfiction committed:
fix the member management button from crashing the program

simonisfiction committed:
Obtain the table index from the correct view

i was previously using the wrong view to obtain the index of the member that the user wants to delete. i switched it to the correct one, and now the feature works as intended

simonisfiction committed:
cledan up code for delete member

bigverl commented:
Visual Error. Clicking delete causes the confirmation message QLabel to display "set text 'delete MEMBER NAME ?' " instead of whatever it's intended to display.

simonisfiction committed:
display the name of the person in the delete message

when you click delete, display a message that says "Delete ------ ?" where the dashes are the selected person's name

simonisfiction commented:
The visual error has been fixed, and the feature is ready for a re test

bigverl commented:
Function works great. Ready for merge

simonisfiction committed:
capitalize the confirm and cancel buttons

simonisfiction committed:
Merge branch 'development' into delete_customer
